lib/taskana-core/src/main/java/pro/taskana/task/internal/TaskTransferrer.java

```diff
@@ -101,11 +101,65 @@ BulkOperationResults<String, TaskanaException> transfer(
         workbasketService.getWorkbasket(destinationWorkbasketKey, destinationDomain).asSummary();
     checkDestinationWorkbasket(destinationWorkbasket);
 
-    return transferMultipleTasks(taskIds, destinationWorkbasket, setTransferFlag);
+    return transferMultipleTasks(taskIds, destinationWorkbasket, null, setTransferFlag);
+  }
+
+  BulkOperationResults<String, TaskanaException> transferTasksWithOwner(
```

Comment:
Do we want to rename this to "transferWithOwner", so it's the same schema as in the Bulk transfer method without the owner?

Reply:
Yes we can do that